CMake/Dependencies/libwebsockets-CMakeLists.txt

```diff
@@ -2,8 +2,6 @@ cmake_minimum_required(VERSION 3.6.3)
 
 project(libwebsocket-download NONE)
 
-SET(PATCH_COMMAND git apply --verbose --ignore-whitespace ${CMAKE_CURRENT_LIST_DIR}/libwebsockets-old-gcc-fix-cast-cmakelists.patch)
```

Comment:
Since we are not longer applying this patch, let's remove the libwebsockets-old-gcc-fix-cast-cmakelists.patch file, and we should remove the already not used libwebsockets-leak-pipe-fix.patch file too.

Reply:
@stefankiesz removed unused patch files as a part of the `libwebsockets` update commit.